packages/gatsby-theme-adr/config/feedOptions.js

```diff
@@ -40,7 +40,7 @@ module.exports = {
               node {
                 frontmatter {
                   title
-                  deck
+                  context
```

Comment:
Shouldn't we also grab `deck` in case a site upgrades but it's still using the old format? Also applies in other places.